edgelet/edgelet-kube/src/lib.rs

```diff
@@ -9,11 +9,12 @@
     clippy::use_self
 )]
 
-mod constants;
+pub mod constants; //todo should be hidden from outside but visible for integration tests
```

Comment:
>todo should be hidden from outside but visible for integration tests [](start = 21, length = 68)

pub(crate) work here?

Reply:
Surprisingly it doesn't. Seems like integration tests are not a part of the crate.